fapolicy_analyzer/tests/test_faprofiler.py

```diff
@@ -246,3 +294,32 @@ def test_validateArgs():
     assert ProfSessionArgsStatus.USER_DOESNT_EXIST in dict_valid_args_return
     assert ProfSessionArgsStatus.EXEC_DOESNT_EXIST in dict_valid_args_return
     assert ProfSessionArgsStatus.PWD_DOESNT_EXIST in dict_valid_args_return
+
+
+def test_validateArgs_relative_exec():
+
+    dictArgs = {
+        "executeText": "Now.sh",
+        "argText": "",
+        "userText": os.getenv("USER"),
+        "dirText": "/tmp",
+        "envText": "PATH=.:${PATH}",
+    }
+
+    # Verify non-existent relative exec path is detected
+    dict_valid_args_return = FaProfSession.validateArgs(dictArgs)
+    assert len(dict_valid_args_return) == 1
+    assert ProfSessionArgsStatus.EXEC_NOT_FOUND in dict_valid_args_return
+
+    # Test w/good args; only OK key is in returned dict
+    dictArgs = {
+        "executeText": "ls",
+        "argText": "",
+        "userText": os.getenv("USER"),
+        "dirText": os.getenv("HOME"),
+        "envText": "",
+    }
+
+    dict_valid_args_return = FaProfSession.validateArgs(dictArgs)
+    assert len(dict_valid_args_return) == 1
+    assert ProfSessionArgsStatus.OK in dict_valid_args_return
```

Comment:
I'd like to see this as 2 different tests. You can use pytest parameterize capabilities for this like below. You should alsos add a set of parameters for this to test a `dict_args` with missing or invalid keys.

```suggestion
@pytest.mark.parametrize(
    "dict_args, expected_status",
    [
        (
             {
                "executeText": "Now.sh",
                "argText": "",
                "userText": os.getenv("USER"),
                "dirText": "/tmp",
                "envText": "PATH=.:${PATH}",
            },
            ProfSessionArgsStatus.EXEC_NOT_FOUND,
        ),
        (
            {
                "executeText": "ls",
                "argText": "",
                "userText": os.getenv("USER"),
                "dirText": os.getenv("HOME"),
                "envText": "",
            },
            ProfSessionArgsStatus.OK,
        ),
    ],
)
def test_validateArgs_relative_exec(dict_args, expected_status):
    dict_valid_args_return = FaProfSession.validateArgs(dict_args)
    assert len(dict_valid_args_return) == 1
    assert expected_status in dict_valid_args_return
```

Reply:
Used the pytest parameterize mechanism, and also added code to detect if received dict's keys match expected dict's keys.